data/json/vehicleparts/vehicle_parts.json

```
@@ -3294,5 +3294,37 @@
     "color": "dark_gray",
     "item": "pirate_flag",
     "breaks_into": [ { "item": "splinter", "count": [ 1, 5 ] }, { "item": "pirate_flag", "count": 1 } ]
+  },
+  {
+    "type": "vehicle_part",
+    "id": "foldable_solar_panel",
+    "name": { "str": "foldable solar panel" },
+    "symbol": "#",
+    "color": "yellow",
+    "broken_symbol": "x",
+    "broken_color": "yellow",
+    "damage_modifier": 10,
+    "durability": 20,
+    "description": "A foldable solar panel.  Will very slowly recharge the vehicle's electrical power when exposed to the sun.  Clouds will slow the recharge speed even more.",
+    "epower": 40,
+    "item": "foldable_solar_panel",
+    "folded_volume": "1 L",
+    "location": "on_roof",
+    "requirements": {
+      "install": { "skills": [ [ "mechanics", 4 ] ], "time": "60 m", "using": [ [ "vehicle_bolt", 1 ] ] },
+      "removal": { "skills": [ [ "mechanics", 2 ] ], "time": "30 m", "using": [ [ "vehicle_bolt", 1 ] ] },
+      "repair": {
+        "skills": [ [ "electronics", 6 ] ],
+        "time": "50 m",
+        "using": [ [ "vehicle_screw", 1 ], [ "foldable_solar_panel", 1 ], [ "soldering_standard", 8 ] ]
```

These are crafting requirements, not items. It's a bit weird that solar panel repair uses `solar_panel` for the name of a crafting requirement that's actually slapping on some solar cells to fix it but...
```suggestion
        "using": [ [ "vehicle_screw", 1 ], [ "solar_panel", 1 ], [ "soldering_standard", 8 ] ]
```